### PR TITLE
Audit article about adding disk space after resizing Windows server: …

### PR DESCRIPTION
**Pull request by RackerKate**
…INFODEV-2750

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1c34d49dc9/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-2d432c11c1/how-to/). :bow:

**Comment by RackerKate** [comment]
@cyrichardson A new one for your review, please.

**Comment by cyrichardson** [comment]
@RackerKate On it in just a bit.

**Comment by cyrichardson** [comment]
@RackerKate Suggested changes here: 
[adding-disk-space-after-resizing-a-windows-server-2012-cloud-server_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2586135/adding-disk-space-after-resizing-a-windows-server-2012-cloud-server_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-1e0be523f1/how-to/)

File: content/cloud-servers/adding-disk-space-after-resizing-a-windows-server-2012-cloud-server.md [hunk 1]
```diff
@@ -1,50 +1,57 @@
...
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2018-11-15'
+last_modified_by: Kate Dougherty
 product: Cloud Servers
 product_url: cloud-servers
 ---
 
-After successfully resizing a Windows Cloud Server, you will need to perform
-some additional steps in order to utilize the new disk space that is
+After you successfully resize a Windows&reg; Cloud Server, you need to perform
```
Comment (chri2547) [inline content/cloud-servers/adding-disk-space-after-resizing-a-windows-server-2012-cloud-server.md]: remove 'successfully' - not needed imo.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-39d3fb7039/how-to/)

**Pull request merged**